code/API_definitions/IoT-device-communication-function-management.yaml

```
@@ -0,0 +1,732 @@
+openapi: 3.0.3
+info:
+  title: CAMARA IoT Device Communication Function Management
+  version: 0.1.0

```

A version other than wip (work in progress) will only be set in a release PR. Cf. https://github.com/camaraproject/ReleaseManagement/blob/main/documentation/API_Release_Guidelines.md

```suggestion
  version: wip
```